library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -25,6 +26,18 @@ internal constructor(
     class Builder {
         private var endpointUrl = DatadogEndpoint.SESSION_REPLAY_US1
         private var privacy = SessionReplayPrivacy.MASK_ALL
+        private var extensionSupport: ExtensionSupport = NoOpExtensionSupport()
+
+        /**
+         * Adds an extension support implementation. This is mostly used when you want to provide
+         * different behaviour of the Session Replay when using other Android UI frameworks
+         * than the default ones.
+         * @see [ExtensionSupport.getCustomViewMappers]
+         */
+        fun addExtensionSupport(extensionSupport: ExtensionSupport): Builder {
```

Comment:
`add` has a connotation of the multiple times usage, so I guess it should be backed by `List`. In the end it is a valid case imo when multiple extensions are added.

Reply:
yes but what I had in mind is to actually leverage this later ...and to back it by a list in case it is really needed. I can change that later without modifying the public api. What do you think ?

Reply:
The idea is that for now I can only see this be using from an App or from an external SR module. In that sense I can only see it apply it from when the configuration is being made and only one `ExtensionSupport` implementation / app or module. Maybe later it will need more but I don't see the case for now.

Reply:
Ok, let's have a TODO + task then for that. For me `addXXX` allows multiple calls, while single call is for the `setXXX`.